includes/templates/single-web-story.php

```diff
@@ -43,9 +43,16 @@
 	<noscript><style amp-boilerplate="">body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
 	<?php /* todo: include custom CSS via PHP */ ?>
 	<style amp-custom>
-		.full-bleed > :first-child {
+		.is-fill > :first-child {
```

Comment:
Not for this PR, but I'm curious, why shouldn't we generate the `<style amp-custom>` (or even the rest of this document) entirely in our JS code?

Reply:
Currently, this template is using WordPress filters and functions to use and display relevant data from the site. Also, previously the `amp-custom` was put together automatically after tree-shaking, combining all the relevant styles. Not knowing if this will be the case in the future as well, `amp-custom` was just added as a quick replacement when separating the Stories plugin from AMP. Basically -- this part has just never been seriously discussed in terms of how to store/generate, it might be good to do that before the first release.

For portability, it would be more convenient for example, if the `amp-story` tag would also be saved together with the content, currently, only the pages inside `amp-story` are saved.

Reply:
Filed #233.